### PR TITLE
Lesson 1

### PR DESCRIPTION
**Pull request by DPASCII**

### CODE REVIEW

File: Goodboy-Oldboy.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  (dogYearFuture - dogYearOfBirth) +
+  " human years old in " +
+  dogYearFuture;
+let dogYears =
+  "Your dog will be " +
+  (dogYearFuture - dogYearOfBirth) * 7 +
+  " dog years old in " +
+  dogYearFuture;
+
+console.log((shouldShowResultInDogYears && dogYears) || humanYears);
```
Comment (carlopascual) [inline Goodboy-Oldboy.js:16]: Make one template string, then replace it with the correct outputs.

Try using https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

File: Housey-pricey.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+let volumeInMeters = 8 * 10 * 10;
+let gardenSizeInM2 = 100;
+housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
```
Comment (carlopascual) [inline Housey-pricey.js:3]: add let keyword here

Comment (carlopascual) [reply Housey-pricey.js:3]: try adding `"use strict;"` on top

**Review by carlopascual — COMMENTED** [review]
nice jobbo

File: Goodboy-Oldboy.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+"use strict";
+const dogYearOfBirth = 2018;
+const dogYearFuture = 2025;
+const shouldShowResultInDogYears = false;
+const humanYears = dogYearFuture - dogYearOfBirth;
+const type = shouldShowResultInDogYears ? `dog` : `human`;
```
Comment (carlopascual) [inline Goodboy-Oldboy.js:6]: Fine to use backtick here, but if there's no `${}`, common practice is to use normal quotations

File: Housey-pricey.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+"use strict";
+let volumeInMeters = 8 * 10 * 10;
+let gardenSizeInM2 = 100;
+let housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300;
+const housePeter = 2500000;
+let priceComparison = housePeter > housePrice ? `over` : `under`;
+let buyer = "Peter";
+
+console.log(`${buyer} is ${priceComparison}paying for the house`);
```
Comment (carlopascual) [inline Housey-pricey.js:9]: missing space between `${priceComparison}` and `paying`, but it's fine lol

**Review by carlopascual — APPROVED** [review]
nice sir galing

**Comment by carlopascual** [comment]
also good to be specific sa commit message, like what did you fix ganun hahaha but it's fine for now

File: Housey-pricey.js (hunk 4 above)
Comment (DPASCII) [reply Housey-pricey.js:9]: overpay and underpay are supposed to be 1 word cer haha

File: Goodboy-Oldboy.js (hunk 3 above)
Comment (DPASCII) [reply Goodboy-Oldboy.js:6]: grape

**Pull request merged**